src/tests/Missions.test.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Provider } from "react-redux";
+import { render } from "@testing-library/react";
+import store from "../Redux/configureStore";
+import Missions from "./../components/Missions";
+import missionsReducer, {
+  showMissions,
+  joinMission,
+  leaveMission,
+} from "../Redux/missions/missions";
```

Comment:
- [ ] Your tests fail because of the imports on lines 4 and 10. `Redux` should be redux. Please fix that.